src/io/flutter/run/FlutterReloadManager.java

```diff
@@ -217,14 +220,19 @@ public void saveAllAndReload(@NotNull FlutterApp app) {
     }
   }
 
-  public void saveAllAndRestart(@NotNull FlutterApp app) {
+  public void saveAllAndRestart(@NotNull FlutterApp app, @NotNull Project project) {
```

Comment:
as a nit, I believe that `FlutterApp` already has a reference to the project

Reply:
Fantastic.  Thanks!